src/components/theme/View/View.jsx

```diff
@@ -240,17 +255,25 @@ class View extends Component {
             <meta
               property="og:image"
               content={toPublicURL(
-                this.props.content.image?.scales?.large?.download,
+                this.props.content.opengraph_image.download ||
+                  this.props.content.image?.scales?.large?.download,
               )}
             />
           )}
-          {this.props.content.description && (
+          {(this.props.content.opengraph_description ||
+            this.props.content.seo_description ||
+            this.props.content.description) && (
             <meta
               property="og:description"
-              content={this.props.content.description}
+              content={
+                this.props.content.opengraph_description ||
+                this.props.content.seo_description ||
+                this.props.content.description
+              }
             />
           )}
-          <meta name="twitter:card" content="summary_large_image">
+          {this.props.content.seo_canonical_url}
```

Comment:
is it correct to place canonical outside any tag in header?

Reply:
no its not. will correct.